.circleci/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM debian:jessie as itms_installer
+
+ENV ITMS_VERSION "1.9.7"
+USER root
+
+WORKDIR /tmp
+
+# Install iTMSTransporter
+ADD https://github.com/janpio/fastlane-ubuntu-foo/raw/master/iTMSTransporter_installer_linux_$ITMS_VERSION.sh .
```

Comment:
Is there a chance for us to move this to the fastlane org, to minimize the external dependencies? Also a 50MB `.sh` file?!

Reply:
cc @janpio, this seems to be your repo 

I think the file is kind of like a self-extracting zip file, except it's a tar archive and sh script at the same time. Pretty weird

Reply:
This file is offered by Apple, but only when logged in into developer.apple.com/account. For my PoC I just downloaded it and added it manually.

Reply:
Because of the possible terms problems the current circle/config.yml does NOT actually install iTMSTransporter for now!